.github/ISSUE_TEMPLATE/bug_report_spell.md

```diff
@@ -0,0 +1,96 @@
+---
+name: 🐛 Bug Report - 🧙‍♂️ Spell
+about: Report any bug or defect regarding a spell.
+labels: 'bug, spell'
+title: '🧙‍♂️ [Bug][Spell] '
```

Comment:
Leaving my two cents
1. Remove the bug label and title
2. Personally I am a fan of the DB/Spell: Core/Spell etc naming for all cases because i personally think it gives a cleaner look.

Reply:
Or if we create a label for the two, "spell" is just too generic, same goes for crestures

Reply:
Thats the whole point of this - preordering, giving visual feedback on the issue list by default

Reply:
And here is a visual comparison of the outome:

![image](https://user-images.githubusercontent.com/1238238/109404325-546cb400-7965-11eb-8b8d-fd78ff25de36.png)

![image](https://user-images.githubusercontent.com/1238238/109404330-5fbfdf80-7965-11eb-9061-5c09f02cde81.png)

As clearly can be seen not everyone follows the guidlines, especially since the system is new at `vmangos` while others fail to name their issue at all.

In the long run you will have everyone comply tho as seen at `cmangos` (I did not implement bugreport subtypes there so its a bit uniform)

![image](https://user-images.githubusercontent.com/1238238/109404369-c513d080-7965-11eb-8dd4-b4866b817432.png)

Reply:
Sure but [bug] is unnecesssry and my point still stands.

I would prefer we manually lable them instead in that case because most of the issues you get in any category are issues in either the DB or Core/Script so if you f.ex only label it as "spell" it will be 1000 times harded for s new contributer to choose an issue incase they only know cpp or db.

In this case generalizing is not good. Neither is adding more issue templates.

Reply:
I'm also against having multiple issue templates, as I pointed in my other comment.

Reply:
I disagree

Reply:
First of all, stop resolving this when it is not resolved.

Second of all, why do you disagree? Generalizing is bad, especially for new contributors. Now there is no easy solution for that except for good old manual labeling, so that's why I don't believe in making them auto label with "creature" "spell" etc. For the actual template, I have no issue with it.

Also, again, labeling with "bug" is very unnecessary especially since an open issue does not mean it is a bug, a lot of open reports are just due to outdated cores and therefore not a bug.

Reply:
Yes, please don't mark a discussion as resolved until it's actually resolved. BTW I marked the other discussion as resolved so we can just talk here. As I just said:

@ulfgebhardt I guess we can give it a try, as long as you promise to take care of keeping all the templates in synch :)

so we can give it a try and see how it goes